app/views/shops/_form.html.haml

```diff
@@ -25,4 +25,23 @@
 			= f.label :email_id
 			= f.email_field :email_id
 			%br/
-			= f.submit(class: "btn btn-primary")
\ No newline at end of file
+			Location:
+			%input#address{:style => "width: 200px", :type => "text"}/
+			#map{:style => "width: 500px; height: 400px;"}
+			= f.hidden_field :latitude
+			= f.hidden_field :longitude
+			:javascript
```

Comment:
Could we move this to an external JS file that would be included in the application.js? Need to have jQuery ready function in that and an if condition to check if the map ID exists, then add the javascript here inside that. Having inline javascript is not best practices. In the application.js they will all be compressed and minified, etc.